SOICbite_gEDA.fp

```diff
@@ -0,0 +1,35 @@
+Element["" "REF  " "" "" 0 0 0 0 0 100 ""]
+(
+Pad[75mil -1.100mm 75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "1" ""]
+Pad[25mil -1.100mm 25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "2" ""]
+Pad[-25mil -1.100mm -25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "3" ""]
+Pad[-75mil -1.100mm -75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "4" ""]
+Pad[-75mil -1.100mm -75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "5" "onsolder"]
+Pad[-25mil -1.100mm -25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "6" "onsolder"]
+Pad[25mil -1.100mm 25mil -0.400mm 0.800mm 20mil 1.000mm "GND" "7" "onsolder"]
+Pad[75mil -1.100mm 75mil -0.400mm 0.800mm 20mil 1.000mm "GND" "8" "onsolder"]
+Pin[-100mil -75mil 0.700mm 20mil 0.903mm 0.700mm "GND" "9" "hole"]
```

Comment:
I can see this is the hole definition. Maybe it's just the colour scheme in my gerber viewer. What does the 0.903mm define and what does the 0.700mm define?